### PR TITLE
New Binance topic

### PR DESCRIPTION
**Pull request by SidVal**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

This topic already exists: https://github.com/topics/binance
But I do not why missing index file, and the logo file, and Wikipedia link, etc.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/binance/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+display_name: Binance
+logo: binance.png
+short_description: Is a cryptocurrency exchange which is the largest exchange in the world in terms of daily trading volume of cryptocurrencies.
```
Comment (smashwilson) [inline topics/binance/index.md:4]: Some minor rewording for brevity.

```suggestion
short_description: The largest cryptocurrency exchange in the world in terms of daily trading volume of cryptocurrencies.
```

**Review by smashwilson — CHANGES_REQUESTED** [review]

File: topics/binance/index.md (hunk 1 above)
Comment (SidVal) [reply topics/binance/index.md:4]: Agree

Comment (smashwilson) [inline topics/binance/index.md:4]: Hrm. Looks like this got lost without being applied?

```suggestion
short_description: The largest cryptocurrency exchange in the world in terms of daily trading volume of cryptocurrencies.
```

**Comment by SidVal** [comment]
LOL, I forgot commit xD

**Review by smashwilson — APPROVED** [review]

**Comment by smashwilson** [comment]
There we go. 🚀 

Thanks again! 🙇🏻

**Pull request merged**